### ISSUE TITLE
How to choose sigma?

### ISSUE THREAD

**Issue by TheSunWillRise** [facebookresearch/label_dp_antipodes#1]
Hello. I don't find any clue of how to choose sigma in neither of your paper (Table 4 doesn't mention it) or the codes. As sigma is important for both utility and privacy, I'm wondering:
1. what are the settings of sigma for the different levels of privacy in Table 2?
2. what are the privacy levels of settings in Table 4?

**Comment by karthikprasad** [comment]
Hello! Thank you for your interest. 

sigma is the per-coordinate standard deviation and is computed following the standard analysis of ε-DP with additive Laplace noise: σ = 2√2/ε.

This is mentioned at the end of section 5.1 in the paper and computed used to compute epsilon at https://github.com/facebookresearch/label_dp_antipodes/blob/main/lib/alibi.py#L73 in the code.

1. ~Table 2~ Table **1** lists the epsilon; sigma can be obtained using the above formula.
2. Table 4 lists the hyper-parameters for the runs listed in ~Table 2~ Table **1** (you can join the two tables on accuracy column).

I'll be closing this now; please feel free to reopen the issue if you have any more questions.

**Comment by karthikprasad** [comment]
@815961618 , my apologies.

I misread table 2 as table 1 in your question and responded incorrectly. I have edited the above response to reflect the correct answer.

Let me answer your question here:
> what are the settings of sigma for the different levels of privacy in Table 2?


| ALIBI | ε = 3 | ε = 4 | ε = 5 | ε = 6 | ε = 8 |
| --- | --- | --- | --- | --- | --- |
| lr | 0.1155 | 0.0211 | 0.037 | 0.0539 | 0.0265 |
| weight decay (x 10^-4) | 3.41 | 9.96 | 25.89 | 22.59 | 40.44 |
